tools/run_test_set.py

```diff
@@ -0,0 +1,50 @@
+import sys
+import os
+import subprocess
+from shutil import copyfile
+import filecmp
+
+# specify your build folder here
+sdl_path = "../sdl_build/bin/"
```

Comment:
At the moment it would probably crash if the specified path was invalid(Since the `sdl_path` depends on the name and location of the build folder)

Since its an optional feature, you could disable the backup/restore feature if the specified path doesn’t exist and print a message to say they can setup a path to use that functionality